### PR TITLE
#139924853 Edited the README to point to our hosted App and put up Coverage and CI status badges.

### PR DESCRIPTION
**Pull request by andela-oadeniran**

### CODE REVIEW

**Comment by bmosigisi** [comment]
@andela-oadeniran good job on changing where the branches point. Solve the conflicts and it'll be ready for merge.

**Comment by andela-oadeniran** [comment]
@andela-bmosigisi We have resolved the conflict and you can check and merge.

File: .travis.yml [hunk 1]
```diff
@@ -65,3 +65,15 @@ script:
 
   # Code coverage
   - coverage report
+
+
+
```
Comment (bmosigisi) [inline .travis.yml]: Too many extra spaces

File: .travis.yml [hunk 2]
```diff
@@ -65,3 +65,15 @@ script:
...
+notifications:
+  slack: andela:l6HtahN5TFwe7CaKTqxjFCrJ
+
+# After the build test with coveralls
+after_success:
+- coveralls
+
+
```
Comment (bmosigisi) [inline .travis.yml:77]: Get rid of extra spaces

Comment (andela-oadeniran) [reply .travis.yml:77]: @andela-bmosigisi I have got rid of the spaces you can review and kindly merge. thanks

Comment (bmosigisi) [reply .travis.yml:77]: @andela-oadeniran Check out the first badge in the Readme
screen shot 2017-02-21 at 13 21 51

Comment (andela-oadeniran) [reply .travis.yml:77]: @andela-bmosigisi yes I fixed that, I set cycle CI not to follow the project too.but it is still building

**Review by bmosigisi — APPROVED** [review]
Looks good.

**Pull request merged**